TestAssets/TestProjects/SimpleDependencies/SimpleDependencies.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>2.0.0-beta-001507-00</RuntimeFrameworkVersion>
```

Comment:
We don't use `*` versioning in our repo because it causes more trouble than it is worth.

Currently, the best way to do this is to use the `UseCurrentRuntimeFrameworkVersion()` in the test, which will inject the correct SharedFramework version into this property.

See https://github.com/dotnet/cli/blob/master/test/dotnet.Tests/PackagedCommandTests.cs#L274-L284 for an example

and https://github.com/dotnet/cli/blob/master/src/Microsoft.DotNet.TestFramework/TestAssetInstance.cs#L142-L157 for the code.